### ISSUE TITLE
Pseudophakia

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10848]
**Preferred term label:**
Pseudophakia

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of an artificial intraocular lens

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by daboe01** [comment]
sorry, i missed 500081
